appconfiguration/azure-appconfiguration-conversion-sample-complete/README.md

```diff
@@ -1,13 +1,3 @@
----
-page_type: sample
-languages:
-- java
-products:
-- azure-app-configuration
-description: "Sample project for Azure Spring Cloud Config Conversion client library"
-urlFragment: "azure-appconfiguration-conversion-sample-complete"
```

Comment:
Why delete it?
Could you please add more description here:
![image](https://user-images.githubusercontent.com/13167207/124843720-1e9ba400-dfc5-11eb-8bde-e60740ec4157.png)

Reply:
because the readme file is empty